composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "2c808292bc67bc7fa11a7bd66a10e71d",
+    "content-hash": "4a95e33fc49d25c30d4dacd6edfed638",
```

Comment:
Wait... how did we **drop** 2 dependencies, yet `composer.lock` kept growing?! O_O

Reply:
PHPUnit requires `phpspec/prophecy` and it requires `phpdocumentor/reflection-docblock` so the dependencies are still there... So the `composer.lock` is rearranged.